.github/workflows/build.yml

```diff
@@ -51,4 +51,35 @@ jobs:
           else
             npm i
           fi
-          npm run build
\ No newline at end of file
+          npm run build
+
+  dependencies-license-check:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout the code
+      uses: actions/checkout@v2.3.4
+      with:
+        fetch-depth: 0
+    - name: Setup Python
+      uses: actions/setup-python@v2
+      with:
+        python-version: '3.7'
+    - name: Get explicit and transitive dependencies
+      run: |
+        pip install -r requirements.txt
+        pip freeze > requirements-all.txt
+    - name: Check python
+      id: license_check_report
+      uses: pilosus/action-pip-license-checker@v0.6.1
+      with:
+        requirements: 'requirements-all.txt'
+        fail: 'Copyleft,Other,Error'
+        exclude: '(visions.*|pyzmq.*|debugpy.*|certifi.*|tqdm.*)'
+        # visions is BSD-4
+        # pyzmq is Revised BSD
+        # debugpy is EPL-2.0
+        # certifi is MPL-2.0
+        # tqdm is MPL-2.0
```

Comment:
1. Add the various versions explicitly so we'll make sure to keep this up to date
2. Add a link to license file for each package, for convenient verification
3. Optional - because I'm not sure if possible, but if it is please do so - is there a way to verify that the excluded files are really in use? (where found) and if not it should fail, just to make sure that items here won't be stale over time.

Reply:
1. Done
2. Done
3. Not easily possible :/